ruff.toml

```diff
@@ -0,0 +1,88 @@
+# automatically generated file, do not modify
+# for ruff version 0.8.1 (or higher)
+# note: 'E241', 'E272', 'E201', 'E221' are ignored on runbot in test files when formating a table like structure (more than two space)
+# some rules present here are not enabled on runbot (yet) but are still adviced to follow when possible : ["B904", "COM812", "EM101", "I001", "TRY002"]
+
+
+target-version = "py310"
+
+[lint]
+preview = true
+select = [
+    "BLE",
+    "C",
+    "COM",
+    "E",
+    "EM",
+    "EXE",
+    "F",
+    "FA",
+    "FLY",
+    "G",
+    "I",
+    "ICN",
+    "INT",
+    "ISC",
+    "LOG",
+    "PGH",
+    "PIE",
+    "PLC",
+    "PLE",
+    "PLW",
+    "PYI",
+    "RUF",
+    "SIM",
+    "SLOT",
+    "T",
+    "TC",
+    "TID",
+    "TRY",
+    "UP",
+    "W",
+    "YTT"
+]
+ignore = [
+    "C408",
+    "C420",
+    "C901",
+    "E266",
+    "E501",
+    "E713",
+    "E741",
+    "EM102",
+    "FA100",
+    "PGH003",
+    "PIE790",
+    "PIE808",
+    "PLC2701",
+    "PLW2901",
+    "RUF001",
+    "RUF005",
+    "RUF012",
+    "RUF021",
+    "RUF100",
+    "SIM102",
+    "SIM108",
+    "SIM117",
+    "TID252",
+    "TRY003",
+    "TRY300",
+    "TRY400",
+    "UP006",
+    "UP007",
```

Comment:
Remove UP006 and UP007? (old type annotations)
I think they were added because not everyone had target-version py310 configured.